packages/redux/src/analytics/middlewares/setUser.ts

```diff
@@ -19,8 +26,9 @@ export const DEFAULT_TRIGGER_ANONYMIZE_ACTION_TYPES = new Set([
 
 // Default user traits picker
 // Will exclude id property only from the generated traits object
-const DEFAULT_USER_TRAITS_PICKER = ({ [USER_ID_PROPERTY]: id, ...rest }) =>
-  rest;
+const DEFAULT_USER_TRAITS_PICKER: (
+  user: Record<string, unknown>,
```

Comment:
I believe we have a `UserData` type on analytics that can be used here. Can you check, please?

Reply:
I think it is better to give a more explicit type instead of a `Record<string, unknown>` as these should be used only on type aliases. For example, this typing is much better:

```js
  user: UserTraits & { id: UserData['id'] },
```